### PR TITLE
14: Fix `CMake` vars

### LINKED ISSUE #14: Fix CMake vars
With https://github.com/DARMA-tasking/vt/pull/2217 merged, we should update CMake args

### PR DESCRIPTION
**Pull request by JacobDomagala**
Fixes #14

### CODE REVIEW

**Review by nmm0 — CHANGES_REQUESTED** [review]
I think these should be conditioned on the version, so that older versions of vt will still work

**Comment by JacobDomagala** [comment]
Yeah, you're totally right. Got tunnel vision on our CI failure.

File: packages/darma-vt/package.py [hunk 1]
```diff
@@ -210,10 +209,25 @@ def cmake_args(self):
             "-DUSE_STD_THREAD={}".format(
                 int(self.spec.variants["use_std_thread"].value)
             ),
-            "-Dvt_build_tests={}".format(int(self.spec.variants["build_tests"].value)),
-            "-Dvt_build_examples={}".format(
-                int(self.spec.variants["build_examples"].value)
-            ),
         ]
 
+        if self.spec.version == Version("develop"):
```
Comment (nmm0) [inline packages/darma-vt/package.py:214]: If we do something like

```py
if self.spec.version > Version("1.3.0"):
```

Does spack make that comparison correctly?

If not we could try something like 
```py
if self.spec.version > Version("1.3.0") or self.spec.version == Version("develop"):
```

The reason for this is that when we (eventually) add our next release we won't have to worry about breaking the spack package.

**Review by nmm0 — APPROVED** [review]
Looks good to me!

File: packages/darma-vt/package.py (hunk 1 above)
Comment (JacobDomagala) [reply packages/darma-vt/package.py]: We should probably also add 1.1.0 and 1.2.0 to list of available versions. We currently only support 1.0.0 and develop:

https://github.com/DARMA-tasking/spack-package/blob/master/packages/darma-vt/package.py#L58-L59

**Review by lifflander — APPROVED** [review]

**Pull request merged**